### ISSUE TITLE
Question about real world inference

### ISSUE THREAD

**Issue by firefly-insky** [OpenDriveLab/UniVLA#43]
We fine-tuned using two samples from real environments, each containing 90 images, and the loss converged well(We did not use proprioceptive states). We performed inference using the training data, but there was a significant error between the inference results and the real values.
Can you tell me what details need to be paid attention to when collecting data?
finetune:
`torchrun --standalone --nnodes 1 --nproc-per-node 2 finetune_realworld.py  --batch_size 2 --grad_accumulation_steps 1 --max_steps 10010 --save_steps 5000  --window_size 10 --run_root_dir "./real-world-log"`

dataset:
<img width="1319" height="956" alt="Image" src="https://github.com/user-attachments/assets/ebf9bdcf-d546-4ef2-830f-1aebd660729e" />

model:
<img width="1267" height="378" alt="Image" src="https://github.com/user-attachments/assets/c8fe90bd-35ef-4196-bb8c-d184c7542739" />

inference results:
<img width="1674" height="812" alt="Image" src="https://github.com/user-attachments/assets/b4891023-964a-4761-ad54-17883a22195c" />
